XpringKit/PayID/PayIDError.swift

```diff
@@ -1,5 +1,5 @@
 /// Errors that originate from PayID.
-public enum PayIDError: Error {
+public enum PayIDError: Equatable, Error {
```

Comment:
`Equatable`, Neat!

Do we not need to implement a custom compare function for this to work?

Reply:
Yeah, Swift is able to synthesize this on enums.